packages/timeline-state-resolver/src/integrations/httpSend/index.ts

```diff
@@ -174,157 +96,112 @@ export class HTTPSendDevice extends DeviceWithState<HTTPSendState, DeviceOptions
 			}
 		}
 
-		await this._commandReceiver(time, cmd as HTTPSendCommandContent, 'makeReady', '')
+		await this.sendCommand({
```

Comment:
Not a new issue, but if the http request fails, this will result in a rejected promise for `sendManualCommand`, instead of an `ActionExecutionResult`. Should it?